azurelinuxagent/common/protocol/wire.py

```diff
@@ -852,7 +852,7 @@ def save_if_not_none(goal_state_property, file_name):
                 self._save_cache(self._goal_state.ext_conf.get_redacted_xml_text(), EXT_CONF_FILE_NAME.format(self._goal_state.incarnation))
 
             if self._extensions_goal_state is not None:
-                self._save_cache(self._extensions_goal_state.get_redacted_vm_settings(), VM_SETTINGS_FILE_NAME.format(self._extensions_goal_state.etag))  # TODO: need to redact settings
+                self._save_cache(self._extensions_goal_state.get_redacted_vm_settings(), VM_SETTINGS_FILE_NAME.format(self._extensions_goal_state.etag))
```

Comment:
This change seems unrelated to the stated PR name. Should it belong here?

Reply:
when i revisited the code for get_redacted_vm_settings i realized this TODO is already done, so removing it here